Ring.shader

```diff
@@ -0,0 +1,143 @@
+// Made with Amplify Shader Editor
```

Comment:
Remove amplify-related comments

Reply:
This shader should not account for scene lighting.  It should be unlit, and only allow for textures or vertex colors for any visual definition.